.travis/publish_docs.sh

```diff
@@ -88,13 +102,21 @@ cp -r target/staging/* "$release_tag"/
 
 printf "${CYAN}Creating redirecting index.html.$RESET \n"
 mv target/build_index.sh "."
-sh ./build_index.sh "$release_tag"
+bash ./build_index.sh "$release_tag"
```

Comment:
Travis CI uses `sh` in one of their examples[1] when talking about custom scripts.

[1] https://docs.travis-ci.com/user/customizing-the-build#Customizing-the-Installation-Step